lib/net/peer/peer.ts

```diff
@@ -37,17 +37,18 @@ export class Peer extends events.EventEmitter {
    * @param {boolean}     [options.inbound] true if peer initiated connection
    * @param {string}      [options.transport] transport name
    * @param {Protocols[]} [options.protocols=[]] supported protocols
-   * @param {Logger}      [options.logger] logger instance
    */
   constructor(options: any) {
     super()
+
+    this.config = options.config
```

Comment:
Why does the line ```this.config = options.config``` precede the options clone expression?

Reply:
Have updated on all occurences (next push).